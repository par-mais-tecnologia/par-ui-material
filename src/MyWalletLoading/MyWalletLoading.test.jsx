import React from 'react'
import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import { MyWalletLoading } from '../index'

configure({ adapter: new Adapter() })

describe('MyWalletLoading', () => {
  it('shallow MyWalletLoading', () => {
    const wrapper = shallow(
      <MyWalletLoading />
    )
    expect(wrapper).toBeTruthy()
  })

  it('shallow MyWalletLoading match Snapshot', () => {
    const wrapper = shallow(
      <MyWalletLoading fontSize='3em' />
    )
    expect(wrapper).toMatchSnapshot()
  })
})
