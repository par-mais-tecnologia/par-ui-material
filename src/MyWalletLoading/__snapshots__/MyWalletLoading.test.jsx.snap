// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyWalletLoading shallow MyWalletLoading match Snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MyWalletLoading
    fontSize="3em"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="flex justify-center items-center"
      >
        <div
          className="container"
          style={
            Object {
              "fontSize": "3em",
            }
          }
        >
          <div
            className="back-layer"
          >
            <p
              className="rounded-elegance"
            >
              <span
                className="hashtag"
              >
                #
              </span>
              <span
                className="cliente"
              >
                Cliente
              </span>
              <span
                className="mais"
              >
                Mais
              </span>
            </p>
          </div>
          <div
            className="front-layer"
          >
            <p
              className="linear-wipe rounded-elegance"
            >
              <span>
                #ClienteMais
              </span>
            </p>
          </div>
        </div>
      </div>,
      "className": "flex justify-center items-center",
      "style": Object {
        "height": "100%",
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="container"
          style={
            Object {
              "fontSize": "3em",
            }
          }
        >
          <div
            className="back-layer"
          >
            <p
              className="rounded-elegance"
            >
              <span
                className="hashtag"
              >
                #
              </span>
              <span
                className="cliente"
              >
                Cliente
              </span>
              <span
                className="mais"
              >
                Mais
              </span>
            </p>
          </div>
          <div
            className="front-layer"
          >
            <p
              className="linear-wipe rounded-elegance"
            >
              <span>
                #ClienteMais
              </span>
            </p>
          </div>
        </div>,
        "className": "flex justify-center items-center",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="back-layer"
            >
              <p
                className="rounded-elegance"
              >
                <span
                  className="hashtag"
                >
                  #
                </span>
                <span
                  className="cliente"
                >
                  Cliente
                </span>
                <span
                  className="mais"
                >
                  Mais
                </span>
              </p>
            </div>,
            <div
              className="front-layer"
            >
              <p
                className="linear-wipe rounded-elegance"
              >
                <span>
                  #ClienteMais
                </span>
              </p>
            </div>,
          ],
          "className": "container",
          "style": Object {
            "fontSize": "3em",
          },
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <p
                className="rounded-elegance"
              >
                <span
                  className="hashtag"
                >
                  #
                </span>
                <span
                  className="cliente"
                >
                  Cliente
                </span>
                <span
                  className="mais"
                >
                  Mais
                </span>
              </p>,
              "className": "back-layer",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <span
                    className="hashtag"
                  >
                    #
                  </span>,
                  <span
                    className="cliente"
                  >
                    Cliente
                  </span>,
                  <span
                    className="mais"
                  >
                    Mais
                  </span>,
                ],
                "className": "rounded-elegance",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "#",
                    "className": "hashtag",
                  },
                  "ref": null,
                  "rendered": "#",
                  "type": "span",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Cliente",
                    "className": "cliente",
                  },
                  "ref": null,
                  "rendered": "Cliente",
                  "type": "span",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Mais",
                    "className": "mais",
                  },
                  "ref": null,
                  "rendered": "Mais",
                  "type": "span",
                },
              ],
              "type": "p",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <p
                className="linear-wipe rounded-elegance"
              >
                <span>
                  #ClienteMais
                </span>
              </p>,
              "className": "front-layer",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <span>
                  #ClienteMais
                </span>,
                "className": "linear-wipe rounded-elegance",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "#ClienteMais",
                },
                "ref": null,
                "rendered": "#ClienteMais",
                "type": "span",
              },
              "type": "p",
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="flex justify-center items-center"
        >
          <div
            className="container"
            style={
              Object {
                "fontSize": "3em",
              }
            }
          >
            <div
              className="back-layer"
            >
              <p
                className="rounded-elegance"
              >
                <span
                  className="hashtag"
                >
                  #
                </span>
                <span
                  className="cliente"
                >
                  Cliente
                </span>
                <span
                  className="mais"
                >
                  Mais
                </span>
              </p>
            </div>
            <div
              className="front-layer"
            >
              <p
                className="linear-wipe rounded-elegance"
              >
                <span>
                  #ClienteMais
                </span>
              </p>
            </div>
          </div>
        </div>,
        "className": "flex justify-center items-center",
        "style": Object {
          "height": "100%",
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="container"
            style={
              Object {
                "fontSize": "3em",
              }
            }
          >
            <div
              className="back-layer"
            >
              <p
                className="rounded-elegance"
              >
                <span
                  className="hashtag"
                >
                  #
                </span>
                <span
                  className="cliente"
                >
                  Cliente
                </span>
                <span
                  className="mais"
                >
                  Mais
                </span>
              </p>
            </div>
            <div
              className="front-layer"
            >
              <p
                className="linear-wipe rounded-elegance"
              >
                <span>
                  #ClienteMais
                </span>
              </p>
            </div>
          </div>,
          "className": "flex justify-center items-center",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="back-layer"
              >
                <p
                  className="rounded-elegance"
                >
                  <span
                    className="hashtag"
                  >
                    #
                  </span>
                  <span
                    className="cliente"
                  >
                    Cliente
                  </span>
                  <span
                    className="mais"
                  >
                    Mais
                  </span>
                </p>
              </div>,
              <div
                className="front-layer"
              >
                <p
                  className="linear-wipe rounded-elegance"
                >
                  <span>
                    #ClienteMais
                  </span>
                </p>
              </div>,
            ],
            "className": "container",
            "style": Object {
              "fontSize": "3em",
            },
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <p
                  className="rounded-elegance"
                >
                  <span
                    className="hashtag"
                  >
                    #
                  </span>
                  <span
                    className="cliente"
                  >
                    Cliente
                  </span>
                  <span
                    className="mais"
                  >
                    Mais
                  </span>
                </p>,
                "className": "back-layer",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <span
                      className="hashtag"
                    >
                      #
                    </span>,
                    <span
                      className="cliente"
                    >
                      Cliente
                    </span>,
                    <span
                      className="mais"
                    >
                      Mais
                    </span>,
                  ],
                  "className": "rounded-elegance",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "#",
                      "className": "hashtag",
                    },
                    "ref": null,
                    "rendered": "#",
                    "type": "span",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Cliente",
                      "className": "cliente",
                    },
                    "ref": null,
                    "rendered": "Cliente",
                    "type": "span",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Mais",
                      "className": "mais",
                    },
                    "ref": null,
                    "rendered": "Mais",
                    "type": "span",
                  },
                ],
                "type": "p",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <p
                  className="linear-wipe rounded-elegance"
                >
                  <span>
                    #ClienteMais
                  </span>
                </p>,
                "className": "front-layer",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <span>
                    #ClienteMais
                  </span>,
                  "className": "linear-wipe rounded-elegance",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "#ClienteMais",
                  },
                  "ref": null,
                  "rendered": "#ClienteMais",
                  "type": "span",
                },
                "type": "p",
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;
